source/_posts/selling-your-solid-oop-principles.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Selling your SOLID OOP principles
+date: 2018-05-23
+author: Hector Yeomans
+draft: true
```

Comment:
I think this is not the right way to do a draft, you need to `hexo new draft title`

Reply:
Ah got it, I don't remember adding this.